templates/snippet.html

```diff
@@ -62,8 +62,6 @@ <h1>{{ model.Details.Title }}</h1>
         <span class="caret"></span>
       </button>
       <ul class="dropdown-menu">
-        <li><a href="http://tsunami.io/cloud_tsunami.html?initurl=http://fssnip.net/raw/{{ model.Details.Id | format_id }}">tsunami.io</a></li>
-        <li><a href="http://www.tryfsharp.org/Create?codeUrl=http://fssnip.net/raw/{{ model.Details.Id | format_id }}">tryfsharp.org</a></li>
         <li><a href="http://tryfs.net/snippets/snippet-{{ model.Details.Id | format_id }}">tryfs.net</a></li>
```

Comment:
This is also broken isn't it?

Reply:
which link is broken?

Reply:
I removed the broken link on the right branch. #104